### PR TITLE
[chip-tool] Add --commissioner-fabricid parameter

### PR DESCRIPTION
**Pull request by vivien-apple**
#### Problem

There is no way to override the fabric id.

#### Change overview
 * Add a `--commissioner-fabricid` optional parameter to cluster commands

### CODE REVIEW

**Review by tcarmelveilleux — APPROVED** [review]

File: examples/chip-tool/commands/common/CHIPCommand.h [hunk 1]
```diff
@@ -62,6 +62,7 @@ class CHIPCommand : public Command
         AddArgument("paa-trust-store-path", &mPaaTrustStorePath);
         AddArgument("commissioner-name", &mCommissionerName);
         AddArgument("commissioner-nodeid", 0, UINT64_MAX, &mCommissionerNodeId);
+        AddArgument("commissioner-fabricid", 0, UINT64_MAX, &mCommissionerNodeId);
```
Comment (msandstedt) [inline examples/chip-tool/commands/common/CHIPCommand.h:65]:
```suggestion
        AddArgument("commissioner-fabricid", 0, UINT64_MAX, & mCommissionerFabricId);
```

Comment (msandstedt) [reply examples/chip-tool/commands/common/CHIPCommand.h:65]: An additional question for my edification: what is the chip-tool cli interface to actually pass this parameter?

Comment (bzbarsky-apple) [reply examples/chip-tool/commands/common/CHIPCommand.h:65]: `--commissioner-fabricid 124345` or `--commissioner-fabricd 0xabcdef1234567890`

**Review by bzbarsky-apple — APPROVED** [review]
Looks good, with @msandstedt's comment address.

**Pull request merged**